.github/workflows/CI.yml

```diff
@@ -31,8 +36,7 @@ jobs:
       - name: Install system packages
         run: |
           sudo apt-get update -qq || true
-          sudo apt-get install -y --no-install-recommends zlib1g-dev gcc g++ gfortran
-          sudo apt-get install -y mpi-default-bin mpi-default-dev build-essential
+          sudo apt-get install -y --no-install-recommends gcc g++ gfortran mpi-default-bin mpi-default-dev
```

Comment:
Trim some fat, and spend less time doing dependency resolution & package installation.